index.html

```diff
@@ -288,13 +319,13 @@ <h2>
             delivered.
           </p>
           <p>
-            A <a>push subscription</a> is <a>deactivated</a> when its associated <a>service worker
-            registration</a> is unregistered, though a <a>push subscription</a> MAY be
-            <a>deactivated</a> earlier.
+            A <a>push subscription</a> without a [=push subscription/window-accessible scope=] is
+            <a>deactivated</a> when its associated <a>service worker registration</a> is
+            unregistered, though a <a>push subscription</a> MAY be <a>deactivated</a> earlier.
```

Comment:
Should we provide a way to migrate to window-accessible scope to make an existing non-root scoped subscription be independent from service worker, probably as a followup?

Reply:
I think that's already possible as once you have permission it's quite easy to get a new subscription in, but maybe that varies across implementations.

Reply:
That should be sufficient for Firefox at least, yup.